TTHAnalysis/python/plotter/susy-sos/sos_plots.py

```diff
@@ -0,0 +1,546 @@
+#!/usr/bin/env python
```

Comment:
@rcastello @cbotta for the future, this should be made either more compact (with less duplication) or implemented using some centralized interface